main.ts

```diff
@@ -1,129 +1,197 @@
 import { initializeGame } from './src/game';
-import { Thousand, Phase, Game, Battle, PlayersBid } from './src/game.interfaces';
+import { Thousand, Phase, Game, Battle, PlayersBid, TrumpAnnouncement, Suit } from './src/game.interfaces';
 import * as _ from 'lodash';
-import { shareStock, throwCard, Bid } from './src/game.actions';
-import { createCards } from './src/helpers/cards.helpers';
+import { shareStock, throwCard, Bid, Action, THROW_CARD, ThrowCard, REGISTER_PLAYER, RegisterPlayer, BID, SHARE_STOCK, ShareStock } from './src/game.actions';
+import { createCards, toString } from './src/helpers/cards.helpers';
+import { getWinner, getNextTurn } from './src/helpers/players.helpers';
+import { getTrickWinner } from './src/helpers/battle.helpers';
 
-const defaultState: Game = {
-    settings: {
-        permitBombOnBarrel: true,
-        maxBombs: 2,
-        barrelPointsLimit: 880
-    },
-    phase: Phase.BATTLE_IN_PROGRESS,
-    players: [
-        {id: 'adam', battlePoints: []},
-        {id: 'alan', battlePoints: []},
-        {id: 'pic', battlePoints: []}
-    ],
-    deck: [],
-    stock: [],
-    bid: [
-        { player: 'adam', bid: 0, pass: true },
-        { player: 'pic', bid: 0, pass: true },
-        { player: 'alan', bid: 110, pass: false },
-        { player: 'adam', bid: 100, pass: false }
-    ] as PlayersBid[],
-    cards: {},
-    battle: {
-        trumpAnnouncements: [],
-        leadPlayer: 'alan',
-        trickCards: [],
-        wonCards: {
-            'adam': [ ...createCards(8) ],
-            'alan': [ ...createCards(8) ],
-            'pic': [ ...createCards(8) ]
-        }
-    } as Battle
-};
-
-const thousand: Thousand = initializeGame(defaultState);
+// const battleFinishedState: Game = {
```

Comment:
I am against commented code. If it is commended it is not needed?

Reply:
I will move it later to another file, main.js is an API showcase, so it's more how to use 1k rather than logic itself